examples/utils/example-utils/src/containerViewRuntimeFactory.ts

```diff
@@ -49,17 +27,30 @@ export class ContainerViewRuntimeFactory<T> extends BaseContainerRuntimeFactory
 		// and add our default view request handler.
 		super({
 			registryEntries: new Map([[dataStoreFactory.type, Promise.resolve(dataStoreFactory)]]),
-			requestHandlers: [
-				// eslint-disable-next-line import/no-deprecated
-				mountableViewRequestHandler(MountableView, [makeViewRequestHandler(viewCallback)]),
-			],
-			provideEntryPoint: async (containerRuntime: IContainerRuntime) => {
-				const entryPoint =
+			provideEntryPoint: async (
+				containerRuntime: IContainerRuntime,
+			): Promise<IFluidMountableViewEntryPoint> => {
+				const entryPointHandle =
 					await containerRuntime.getAliasedDataStoreEntryPoint(dataStoreId);
-				if (entryPoint === undefined) {
-					throw new Error("default dataStore must exist");
+
+				if (entryPointHandle === undefined) {
+					throw new Error(`Default dataStore [${dataStoreId}] must exist`);
+				}
+
+				const entryPoint = await entryPointHandle.get();
+
+				const view = viewCallback(entryPoint as T);
+				// eslint-disable-next-line @typescript-eslint/no-unsafe-return
+				let getDefaultView = async () => view;
+				if (MountableView.canMount(view)) {
+					getDefaultView = async () => new MountableView(view);
```

Comment:
I would expect `getDefaultView` to just return `view`, distinguishing it from `getDefaultMountableView`.

The extra wrapper and copy of React is only necessary for crossing bundle boundaries, so it's nice to have both options.